DoSomething/intertwine
# Importer Basics

#### What's this PR do?

Provides foundation for importing the SMS game json config files and converting them into Twine stories. Currently only supports the story config passage and regular DS passages.

Also some cleanup with the project name and fields in the package.json.
#### Where should the reviewer start?
###### storylistview.js

The file being imported is read here. I added pulling the filename and extension in order to know if it should be processed as one of our .json config files instead.
###### app.js

Basically just acts as a bridge to importsms. Seems like the only reason I have this is to stay consistent with how the original `app.importFile` worked.
###### importsms.js

Reads through all a config object's keys and creates corresponding passages to display in Twine.
#### How should this be manually tested?

I tested with this file: https://gist.github.com/jonuy/88a949644e3eb437da0e

Import it using the "Import From File" button on the Intertwine homepage. Verify that 4 passages get created with data and are connected.

@tongxiang
